trl/trainer/sft_trainer.py

```diff
@@ -304,8 +294,10 @@ def make_inputs_require_grad(module, input, output):
 
         if processing_class.padding_side is not None and processing_class.padding_side != "right":
             warnings.warn(
-                "You passed a processing_class with `padding_side` not equal to `right` to the SFTTrainer. This might lead to some unexpected behaviour due to "
-                "overflow issues when training a model in half-precision. You might consider adding `processing_class.padding_side = 'right'` to your code."
+                "You passed a processing_class with `padding_side` not equal to `right` to the SFTTrainer. This might "
+                "lead to some unexpected behaviour due to overflow issues when training a model in half-precision. "
+                "You might consider adding `processing_class.padding_side = 'right'` to your code.",
+                UserWarning,
```

Comment:
Warnings must be actionable.
Use the appropriate warning type.


This warning is still not actionable. Any idea?

Reply:
Are we in the "Correct but very likely a mistake → Warning with option to disable" case?